api/src/main/java/org/pmiops/workbench/cdr/model/Criteria.java

```diff
@@ -0,0 +1,235 @@
+package org.pmiops.workbench.cdr.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name = "criteria", catalog="cdr")
```

Comment:
So "catalog" determines what DB it goes into?

You may want to add a TODO / comment somewhere mentioning that in future we'll have a dynamic way of specifying what DB to use here.

Reply:
For some odd reason.. the schema attribute(worked on test cases, but not when deployed) didn't work here. I had to use catalog attribute instead. I will add comments /TODO for this